Java/small_game/Game.java

```diff
@@ -0,0 +1,122 @@
+/*
+        Simple program based on basics of java
+        i.e. Input/Output and simple decision making
+        with very well formatted Output
+*/
+import java.util.Scanner;
+public class Game
+{
+    public static void main(String args[])
+    {
+        Scanner in=new Scanner(System.in);
+        System.out.println("==========================================================");
+        System.out.println("welcome to WHO WHANT TO BE TOKEN KING");
+        System.out.println("==========================================================");
+        System.out.print("enter your name : ");
+        String name;
+        name=in.nextLine(); // Name of Player is inputted here
+        System.out.println("==========================================================");
+        System.out.println();
+        System.out.println();
+        System.out.println();
+        int s=0;
+        System.out.println("%%%%%%%%Instruction are as follows%%%%%%%%%");// Instruction for Playing
+        System.out.println("1.You should not do any cheating");
+        System.out.println("2.Five question will be asked answer them by giving its respective  letter");
+        System.out.println("3.If you give all 5 questions correctly u will be given ur token back");
+        System.out.println("==========================================================");
+        System.out.println("==========================================================");
+
+        System.out.println();
+        System.out.println("1.How many members are nominated by president in lok sabha in india"); // Question 1

```

Comment:
Rather than hardcoding the questions, create a function that reads all the questions from a text file (with options). Then let the user input all the answers one by one and as the user enters the answer, store it in an array or any other DS you prefer. It should continue till the last question and then for the score, compare it with the actual answers stored in another text file. 

This makes it super easy to modify, add, delete, or update questions.

Reply:
I don't have any idea about file handling in java thus it will be bit hard for me to incorporate this change but I assure you that all other changes that are needed will be done as soon as possible

Reply:
Have a look over the internet about this, it's not a big thing. Have a look at this:
https://www.geeksforgeeks.org/file-handling-java-using-filewriter-filereader/

Reply:
@kaustubhgupta okay i will try to do the said changes after going through this article
and thank you for help